docs/docusaurus/versioned_docs/version-1.7.0/orc8r/upgrade_1_5.md

```diff
@@ -66,7 +66,7 @@ This process prepares your cluster for Orc8r-NMS DB unification. (In release 1.5
 *Paste in shell prompt:*
 
 ```bash
-wget https://raw.githubusercontent.com/magma/magma/master/nms/packages/magmalte/scripts/fuji-upgrade/pre-upgrade-migration.sh && chmod +x pre-upgrade-migration.sh && ./pre-upgrade-migration.sh
+wget https://raw.githubusercontent.com/magma/magma/v1.5/nms/app/packages/magmalte/scripts/fuji-upgrade/pre-upgrade-migration.sh && chmod +x pre-upgrade-migration.sh && ./pre-upgrade-migration.sh
```

Comment:
Typo here, this looks like it should be pointing to v1.7 branch

Reply:
Same as about, I wasn't sure what the best strategy is here is.

Reply:
Ignore my comment here. Assume same response here as what I replied with elsewhere.